clippy_lints/src/blocks_in_if_conditions.rs

```diff
@@ -29,7 +29,7 @@ declare_clippy_lint! {
     /// if true { /* ... */ }
     /// ```
     ///
-    /// // or
+    /// or
```

Comment:
In this example, I would agree. There might be some cases where this could be overwhelming, though. Could you review the other "// or" cases as well? If it's the same story, then we can just remove the "// bad" comments and separate the examples into the two blocks. :)

Reply:
All of the other occurrences look alright if they were to be combined. It doesn't seem to be that common of a pattern, and when it's used the examples are pretty short.

Reply:
Perfect. Can you combine them in this PR?

Reply:
Done. Can I replace the `// Good`/`// Bad` comments in another PR? There are too many to do at once :smile:.